### PR TITLE
Correct grammar in docs

### PR DESCRIPTION
**Pull request by shivdhar**

### CODE REVIEW

File: gensim/scripts/glove2word2vec.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 Notes
 -----
 
-GloVe format (real example can be founded `on Stanford size <https://nlp.stanford.edu/projects/glove/>`_) ::
+GloVe format (real example can be found `on Stanford site <https://nlp.stanford.edu/projects/glove/>`_) ::
```
Comment (mpenkov) [inline gensim/scripts/glove2word2vec.py]:
```suggestion
GloVe format (real example can be found on the `Stanford site <https://nlp.stanford.edu/projects/glove/>`_) ::
```

File: gensim/scripts/glove2word2vec.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
     word9 0.334 0.241 0.324 0.188
 
 
-Word2Vec format (real example can be founded `on w2v old repository <https://code.google.com/archive/p/word2vec/>`_) ::
+Word2Vec format (real example can be found `on w2v old repository <https://code.google.com/archive/p/word2vec/>`_) ::
```
Comment (mpenkov) [inline gensim/scripts/glove2word2vec.py]:
```suggestion
Word2Vec format (real example can be found in the `old w2v repository <https://code.google.com/archive/p/word2vec/>`_) ::
```

File: gensim/scripts/glove2word2vec.py (hunk 1 above)
Comment (piskvorky) [inline gensim/scripts/glove2word2vec.py]:
```suggestion
GloVe format (a real example can be found on the `Stanford site <https://nlp.stanford.edu/projects/glove/>`_) ::
```

File: gensim/scripts/glove2word2vec.py (hunk 2 above)
Comment (piskvorky) [inline gensim/scripts/glove2word2vec.py]:
```suggestion
Word2Vec format (a real example can be found in the `old w2v repository <https://code.google.com/archive/p/word2vec/>`_) ::
```

**Comment by mpenkov** [comment]
Hi @shivdhar, could you please apply the suggestions so we can merge the PR?

**Comment by shivdhar** [comment]
Done, sorry for the delay!

**Comment by mpenkov** [comment]
Your changes fail flake8, can you please fix?

    gensim/scripts/glove2word2vec.py:25:121: E501 line too long (123 > 120 characters)

**Comment by shivdhar** [comment]
Not sure how to do that; I have allowed edits from maintainers though, so you can make any changes you like.

File: gensim/scripts/glove2word2vec.py [hunk 3]
```diff
@@ -13,7 +13,8 @@
 Notes
 -----
 
-GloVe format (real example can be founded `on Stanford size <https://nlp.stanford.edu/projects/glove/>`_) ::
+GloVe format (real example can be found on the
```
Comment (mpenkov) [inline gensim/scripts/glove2word2vec.py:16]:
```suggestion
GloVe format (a real example can be found on the
```

**Comment by mpenkov** [comment]
Merging. Sorry it took us so long to get around to it. Thank you for your contribution @shivdhar! Congrats on the first PR to gensim :1st_place_medal:

**Comment by shivdhar** [comment]
Thanks, glad to make a (very) small contribution to this wonderful library!

**Pull request merged**